engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
```

Comment:
I was thinking that copying between two locations in the same google cloud bucket would be faster than sending from a VM to the bucket. I'm not sure whether this would be a significant change, given that all of the files are on Google's cloud regardless.